src/huntsman/pocs/autoflats.py

```diff
@@ -0,0 +1,428 @@
+import os
+from contextlib import suppress
+from collections import abc, defaultdict
+from concurrent.futures import wait as wait_for_futures
+
+import numpy as np
+from astropy.stats import sigma_clipped_stats
+from astropy import units as u
+from astropy.io import fits
+
+from panoptes.utils.time import current_time, wait_for_events
+from panoptes.utils import get_quantity_value
+from huntsman.pocs.utils.logger import logger as LOGGER
+
+
+class AutoFlatFieldSequence():
+    """ Class to facilitate flat fields with automatic exposure time updates.
```

Comment:
```suggestion
    """ Class to facilitate taking flat fields with automatic exposure time determination using current sky brightness. Works both for morning and evening twilights or even dome flats.
```

Reply:
That's a bit too long for opening line but could be moved into docstring.